### DISCUSSION TITLE
Save Automation in Cloud Budibase

### DISCUSSION THREAD

**Discussion by satriowicaksn** [Budibase/budibase#3800] [Help]
I'm trying to create Automation in Cloud Budibase with Triggers : Row Created on my table. 
I run test on this Automation and it's run successfully. But, when I'm try to manually insert row in my table, this trigger not run. Why this is happen ? 
I see on : https://docs.budibase.com/building-apps/automate/activating-and-testing that "All automations will start initially in a inactive state, meaning that they will not attempt to process anything until you are ready for them to do so.
To active an automation simply click the big play button just next to the Save Automation button, the automation will now be active in your builder."

I'm try to find this save automation button on my cloud budibase, but i'cant find it. 
![image](https://user-images.githubusercontent.com/90515555/146301959-8dee3d8e-91b2-488c-873f-dd4a6511f74c.png)

**Comment by mjashanks** [comment]
Hi Satrio. Or docs are out of date there.

We no longer have an active function. We changed it so that automations run on published apps only. So, as soon as you publish your app, the automation should be active. 

Similarly, when you change an automation, the changed items don't take effect until the app is published.
